src/ripple/app/main/Application.cpp

```diff
@@ -257,6 +258,7 @@ class ApplicationImp
     NodeStoreScheduler m_nodeStoreScheduler;
     std::unique_ptr <SHAMapStore> m_shaMapStore;
     std::unique_ptr <NodeStore::Database> m_nodeStore;
+    PendingSaves pendingSaves_;
```

Comment:
@seelabs veto - this lets you see what was changed recently versus what is old. I rather have additional information instead of adherance to conformity for conformity's sake.